areas/storage/src/AzureMcp.Storage/Commands/Blob/BlobUploadCommand.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Core.Commands;
+using AzureMcp.Storage.Models;
+using AzureMcp.Storage.Options;
+using AzureMcp.Storage.Options.Blob;
+using AzureMcp.Storage.Services;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Storage.Commands.Blob;
+
+public sealed class BlobUploadCommand(ILogger<BlobUploadCommand> logger) : BaseBlobCommand<BlobUploadOptions>
+{
+    private const string CommandTitle = "Upload File to Blob";
```

Comment:
Nit: would you like to add "Local" to the title so it's clear the file is not referring to Azure File Share?

Reply:
Yeah, I'll update the title to make this a bit more descriptive, especially since "File -> Blob" does work.